SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/SecureVault/AutofillDatabaseProvider.swift

```diff
@@ -1113,6 +1191,98 @@ extension DefaultAutofillDatabaseProvider {
         }
     }
 
+    static func migrateV14(database: Database) throws {
```

Comment:
Identities support for Sync will be a separate PR but included the required DB migration code here for ease of testing later